doc/modules/ROOT/pages/index.adoc

```diff
@@ -55,7 +55,6 @@ public class HazelcastAdditionalSettings
 
     public HazelcastMemberConfiguration onCreated(BeanCreatedEvent<HazelcastMemberConfiguration> event) {
         HazelcastMemberConfiguration configuration = event.getBean();
-        configuration.getGroupConfig().setName("micronaut-cluster");
```

Comment:
Why do we need to remove this? In k8s I got the reason but here this should not be a problem.

Reply:
![image](https://user-images.githubusercontent.com/44066377/88669898-c1288300-d0e4-11ea-9d0b-c0cdcdbbcf1b.png)
 
I ran it again now with that line, it gave this error.

Reply:
getGroupConfig().setName(String) belongs to Hazelcast 3.x. Now that you have upgraded it to 4.x, please try `configuration.setClusterName("micronaut-cluster");`

Reply:
Ah I see, trying it now.